packages/okta-vue/src/components/ImplicitCallback.vue

```diff
@@ -0,0 +1,10 @@
+<template></template>
```

Comment:
Is it necessary to define an entire component for this callback handler?  It appears you can use an empty render() function to bypass this need.

Reply:
Good find. I've updated the export function. 👍